ogc-rs/src/lib.rs

```diff
@@ -1,20 +1,6 @@
 //! # ogc-rs
 //!
 //! ``ogc-rs`` is a safe, idiomatic wrapper around ``ogc-sys``.
-//!
-//! ``ogc-rs`` provides many features from libogc such as:
-//!
-//! * ``network``: Provides TCP networking for the Wii.
-//! * ``audio``: Provides functions for audio on the Wii.
-//! * ``fs``: Provides functions for manipulating the filesystem on the Wii.
-//! * ``system``: Provides OS functions for the Wii.
-//! * ``console``: Provides console functions for the Wii.
-//! * ``input``: Provides an interface for reading input from devices on the Wii.
-//! * ``video``: Provides functions for video output on the Wii.
-//! * ``gx``: Provides an opengl-like interface for rendering on the Wii.
-//!
-//! ``ogc-rs`` also provides runtime functions and an allocator for ``no_std``
-//! environments.
```

Comment:
This can be nice for people who need to know what file provides what from the original library, any reason to delete it?

Reply:
I don't remember why I removed them earlier. Reverted.